static-site/src/pages/whoami.jsx

```diff
@@ -12,17 +12,30 @@ const UserPage = () => {
     <Fragment>
       You&apos;re logged in as <strong>{user.username}</strong>.
       <SubText>
-        You have access to the following private Nextstrain groups, which each
+        You have access to the following Nextstrain groups, which each
```

Comment:
+1 for listing user membership of public groups. 

Slight devil's advocate of whether this will confuse users because technically they can access _all_ public groups via nextstrain.org.

Reply:
Updated wording in #780:

> You are a member of the following Nextstrain groups, which each …